src/test/java/com/zepben/evolve/services/network/testdata/FillFields.kt

```diff
@@ -121,6 +121,14 @@ private fun IdentifiedObject.fillFields() {
     name = "1"
     description = "the description"
     numDiagramObjects = 2
+
+    for (i in 0..1) {
+        val nameType = NameType("name type $i").apply {
```

Comment:
This name type should be added to the service. Surprised the service isn't being passed here already, but it's going to need to.